parm/atm/obs/config/amsua_metop-a.yaml

```diff
@@ -0,0 +1,484 @@
+obs space:
+  name: amsua_metop-a
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: $(DATA)/obs/$(OPREFIX)amsua_metop-a.{{ current_cycle | to_YMDH }}.nc4
```

Comment:
prepatmiodaobs creates ioda format dump files using a different naming convention.  prepatmiodaobs will create `$(OPREFIX)amsua_metop-a.tm00.nc` for amsua_metop-a.  

The same naming convention comment applies to the dump `obsfile` in other amsua yamls in this PR.

Reply:
Got it.  The convention used above is old.  I will update it
We have atms_npp and atms_n20 validated in the global workflow. I will modify AMSU-A to be consistent with the ATMS.